omniscidb/IR/Expr.h

```diff
@@ -801,8 +807,9 @@ class AggExpr : public Expr {
   AggType agg_type_;  // aggregate type: kAvg, kMin, kMax, kSum, kCount
   ExprPtr arg_;       // argument to aggregate
   bool is_distinct_;  // true only if it is for COUNT(DISTINCT x)
-  // APPROX_COUNT_DISTINCT error_rate, APPROX_QUANTILE quantile
-  std::shared_ptr<const Constant> arg1_;
+  // APPROX_COUNT_DISTINCT error_rate, APPROX_QUANTILE quantile,
+  // CORR second arg
+  ExprPtr arg1_;
```

Comment:
I realize we're counting from 0, but I find the 1 confusing for something that is second :)